### PR TITLE
fax update

### PR DESCRIPTION
**Pull request by mcfatelin**
The main modifications are

1) Adding one configuration for simulation: gauss_noise_sigmas. This is a list of baseline fluctuations on each PMT channel. If "gauss_noise_sigmas" is None, fax will use "gauss_noise_sigma" as an average baseline fluctuation over all PMT channels for waveform simulation.

2) Adding the g4_id ( eventid in input G4 root file) into fax_truth file. This is what we discussed in Tuesday's telecon for helping MC-data comparison. If the input is csv file, the g4_id is -1.

### CODE REVIEW

File: pax/plugins/io/WaveformSimulator.py [hunk 1]
```diff
@@ -261,6 +264,7 @@ def startup(self):
         instruction_number = 0
         instruction = []
         for p in self.instruction_reader:
+            p['g4_id']=-1# create fake g4_id=-1 for csv input
```
Comment (tunnell) [inline pax/plugins/io/WaveformSimulator.py]: __[flake8]__
* [E225] missing whitespace around operator
* [E261] at least two spaces before inline comment

File: pax/plugins/io/WaveformSimulator.py [hunk 2]
```diff
@@ -261,6 +264,7 @@ def startup(self):
         instruction_number = 0
         instruction = []
         for p in self.instruction_reader:
+            p['g4_id']=-1  # create fake g4_id=-1 for csv input
```
Comment (tunnell) [inline pax/plugins/io/WaveformSimulator.py]: __[flake8]__
* [E225] missing whitespace around operator

**Comment by mcfatelin** [comment]
Does anybody know what these compiling errors are about?

**Comment by tunnell** [comment]
Maybe an issue with the plotting library.  Can you try changing this line https://github.com/XENON1T/pax/blob/master/.travis.yml#L73

such that

matplotlib=1.5.1

**Comment by mcfatelin** [comment]
Can we merge this request?

**Comment by tunnell** [comment]
@JelleAalbers objections?

**Comment by JelleAalbers** [comment]
Not at all, thanks for implementing this Qing!

**Pull request merged**